src/commands/Bunnies.js

```diff
@@ -0,0 +1,48 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+
+class Bunnies extends Command {
+    constructor(...args) {
+        super(...args);
+
+        this.aliases      = ['bunny', 'bunnies'];
+        this.module       = 'Fun';
+        this.description  = 'Random Adorable Bunnies';
+        this.usage        = 'bunny';
+        this.example      = 'bunny';
+        this.cooldown     = 7500;
+        this.expectedArgs = 0;
+    }
+
+    async execute({ message }) {
+		const errorText = `Error: ${this.config.emojis.rabbit || '🐰'} No bunnies found.`;
+        try {
+			const utils = this.utils;
+			const responses = [
+				{ search: 'Looking for a bunnies...', found: 'Found one!' },
+			];
+
+			const response = responses[utils.getRandomInt(0, responses.length - 1)];

```

Comment:
This doesn't look like it's needed, since we only have a single response

Reply:
fixed